cloudbuild-staging.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+timeout: 7200s
+
+substitutions:
+  _IMAGE_NAME: team1.task1
+
+steps:
+
+# Build image using Dockerfile in current dir
+# The image is tagged with the commit SHA.
+- name: gcr.io/cloud-builders/docker
+  id: 'build-image'
+  args: [ 'build', '-t', 'gcr.io/${PROJECT_ID}/${_IMAGE_NAME}:${COMMIT_SHA}', '.' ]
+  timeout: 1200s
+
+# Push image to repo
+- name: gcr.io/cloud-builders/docker
+  id: 'push-image-to-repo'
+  args: ['push', 'gcr.io/${PROJECT_ID}/${_IMAGE_NAME}:${COMMIT_SHA}']
+
+# Test all notebooks under test_notebooks dir
+# Use 'breakout syntax' for readability. 
+# https://medium.com/@davidstanke/mastering-google-cloud-build-config-syntax-8c3024607daf
+- name: docker.io/library/python:3.7
+  id: 'test-image'
+  entrypoint: /bin/bash
+  args: 
+    - -c
+    - |
+      pip install gcloud-notebook-training && \
```

Comment:
This is good for the first version.
In the next version we have to use the Notebook Executor API